docs/AnaplanSink-sparksink.md

```diff
@@ -22,3 +24,24 @@
 
  **Password:** The password for authentication. It is suggested to use a secure macro to manage the password. 
  Reference: [Provide secure information to pipelines](https://datafusion.atlassian.net/wiki/spaces/KB/pages/32276556/Provide+secure+information+to+pipelines)
+
+Data Type Mappings from CDAP to Anaplan
+----------
+The following table lists out different CDAP data types, as well as the 
+corresponding Anaplan data type for each CDAP type.
+
+| CDAP type      | Anaplan type  |
+|----------------|---------------|
+| string         | text          |
+| boolean        | bool          |
+| date           | date          |
+| decimal        | numeric       |
+| double / float | numeric       |
```

Comment:
nit: put double, float, int, and long on separate rows.

Also, would be good to arrange the supported and unsupported ones alphabetically (boolean, date, decimal, ..., array, bytes, enum, ...)

Reply:
Got, updated, thanks!